Dockerfile

```diff
@@ -13,6 +13,9 @@ RUN apt-get -qq update && apt-get -qq -y install openssl curl \
 # Deploy application code from builder container
 COPY --from=builder /app/_build/prod/rel/epoch /home/epoch/node
 
+# Deploy TestNet user configuration template
+COPY --from=builder /app/apps/aeutils/test/data/epoch_testnet.yaml /home/epoch/node/epoch_testnet.yaml
```

Comment:
I'm afraid this is suboptimal for this kind of change.
This is adding new layer, see [docker docs](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#minimize-the-number-of-layers) on the subject.

The example config should be part of the documentation, and the documentation should be part of the build artifact.

What we're doing here makes no sense to me, we renamed the example file to "test data", now we're using the "test data" as configuration template.

@ThomasArts as the original initiator for this change, could we somehow resolve this issue in a better way ?